io/disk/fiotest.py.data/fio.yaml

```diff
@@ -0,0 +1,12 @@
+parameters:
+    disk: '/dev/sdd'  
+    dir: '/mnt'
+    fio_job: 'fio-simple.job'
+    fio_tool_url:
```

Comment:
can you please point fio_tool_url as well 

ex:    ` fio_tool_url:  'http://brick.kernel.dk/snaps/fio-2.1.10.tar.gz'` it help  where to edit. as  well when any body try to run using default yaml file it should fail